ts/backend/src/app/features/services/auth-service.spec.ts

```diff
@@ -0,0 +1,106 @@
+import type { Request } from 'express'
+import jwt from 'jsonwebtoken'
+import { JwksClient } from 'jwks-rsa'
+import { beforeAll, describe, expect, test, vi } from 'vitest'
+import { defaults } from '../config/defaults.mjs'
+import { AuthService } from './auth-service.mjs'
+
+const AUTHSERVICE_TIMEOUT = 2 * 1000 // ms
+
+describe('Auth Service', () => {
+  // using a string as secret results in symmetrical algorithm being used, i.e.
```

Comment:
```suggestion
  // using a string as secret results in symmetric algorithm being used, i.e.
```

Reply:
Do we use symmetric key encryption in testing just out of simplicity? For the what we want to test here, that's fine, just wondering.